rs/boundary_node/ic_boundary/src/core.rs

```diff
@@ -345,6 +374,80 @@ pub async fn main(cli: Cli) -> Result<(), Error> {
         vec![Box::new(metrics_runner), Box::new(generic_limiter_runner)];
     runners.append(&mut registry_runners);
 
+    // HTTP Logs Anonymization
+    let tracker = (cli.misc.crypto_config)
+        .zip(cli.obs.obs_log_anonymization_canister_id)
+        .map(|(crypto_config, log_anonymization_cid)| {
+            let c = Arc::new(CryptoComponent::new(
+                &crypto_config,                          // config
+                Some(tokio::runtime::Handle::current()), // tokio_runtime_handle
```

Comment:
Could you point me to the place where the containing tokio runtime is instantiated?

Reply:
👍